x/oracle/types/params.go

```diff
@@ -25,8 +25,8 @@ var (
 
 // Default parameter values
 const (
-	DefaultVotePeriod  = utils.BlocksPerMinute / 2 // 30 seconds
-	DefaultSlashWindow = utils.BlocksPerWeek       // window for a week
+	DefaultVotePeriod  = 1                   // Voting every block
```

Comment:
This is the default we're using in all the chains, I think we should just update it